app/js/actions/careSetting.js

```diff
@@ -6,17 +6,17 @@ import axiosInstance from '../config';
 import networkError from './networkError';
 import loading from './loading';
 
-const fetchPatientCareSettingActionCreator = patientCareSetting => ({
+export const fetchPatientCareSettingActionCreator = patientCareSetting => ({
```

Comment:
Looks good, though I have a recommendation, what if you do all exports once at the end of the file, imagine when many action creators are added and the export word is repeating itself.

Reply:
@hadijahkyampeire, thank you for your observation. I think it is a matter of preference and consistency with the rest of the files in the project.

Reply:
ok